pkg/kubelet/pluginmanager/cache/actual_state_of_world.go

```diff
@@ -75,6 +75,18 @@ type actualStateOfWorld struct {
 
 var _ ActualStateOfWorld = &actualStateOfWorld{}
 
+// NamedPluginHandler holds information for handler and the Name of the plugin
+type NamedPluginHandler struct {
+	Handler PluginHandler
+	Name    string
+}
+
+// SocketPluginHandlers contains the map from socket path to NamedPluginHandler
+type SocketPluginHandlers struct {
```

Comment:
Can we use `sync.Map` here to store the map of socket path -> NamedPluginHandler, instead of creating a struct?

Reply:
It seems both mutex-guarded map and sync.Map are used in the codebase.
If the use of sync.Map is not due to performance concern, I prefer to keep the current formation.